.jenkins

```diff
@@ -405,8 +405,6 @@ pipeline {
                                 -DKokkos_ENABLE_TESTS=ON \
                                 -DKokkos_ENABLE_OPENMP=ON \
                                 -DKokkos_ENABLE_LIBDL=OFF \
-                                -DKokkos_ENABLE_LIBQUADMATH=ON \
-                                -DCMAKE_PREFIX_PATH=/usr/local/lib/gcc/x86_64-unknown-linux-gnu/5.3.0 \
```

Comment:
I understand the prefix is invalid but can it be updated so we can continue testing GCC libquadmath support?

Reply:
The `-DCMAKE_PREFIX_PATH=...` was crouches to help finding libquadmath
https://github.com/kokkos/kokkos/blob/ae5fc649ef4b62b48a01123759ed066bff227b43/cmake/tpls/FindTPLquadmath.cmake#L44-L45

Reply:
The reason I turned it off is because libquadmath on Ubuntu 20.04 is based on gcc 10. There is already gcc 5 and 9 in the image. I didn't want to increase the risk of getting a path wrong by adding one more gcc version. Also I am not sure how libquadmath for gcc 10 will interact with gcc 5.

Reply:
My understanding is that while that library is versioned and released with GCC, it maintains the same kind of strict ABI compatibility standards as libstdc++. If suggest you Try the packaged version and see what you get

Reply:
Also, with the way Ubuntu packages things (inherited from Debian), it should be possible to install *just* libquadmath and nothing else from gcc 10

Reply:
> Also, with the way Ubuntu packages things (inherited from Debian), it should be possible to install just libquadmath and nothing else from gcc 10

No gcc 10 is a dependency of libquadmath. Installing liquadmath, pulls gcc-10

Reply:
I'm only seeing a dependency on gcc-10-base, whose contents are negligible
https://packages.ubuntu.com/focal/amd64/gcc-10-base/filelist